tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -331,4 +404,9 @@ Object.Base {
 		source	"copier.SSP.6.1"
 		sink	"copier.host.5.1"
 	}
+
+	route."6" {
+		source	"copier.module.14.2"
+		sink	"gain.16.1"
+	}
```

Comment:
Looking at the topology, there's no real difference between the two PCMS.

Please try to align with the nocodec topology and a 2-level hierarchy.

![sof-tgl-nocodec (3)](https://user-images.githubusercontent.com/1149035/218623741-2314f80e-5499-4104-8e57-e2be2f935562.png)

Reply:
> Looking at the topology, there's no real difference between the two PCMS.
> 
> Please try to align with the nocodec topology and a 2-level hierarchy.
> 
> ![sof-tgl-nocodec (3)](https://user-images.githubusercontent.com/1149035/218623741-2314f80e-5499-4104-8e57-e2be2f935562.png)

@plbossart , earlier had a discussion with customer. At first instance they want to adopt single level copier since they are working CRAS level changes for this mode for now. Once they have the thirday party algo's they will  migrate to the 2-level hierarchy.